app/controllers/concerns/defendant_helpers.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module DefendantHelpers
+  extend ActiveSupport::Concern
+
+  def defendant
+    @defendant ||= @search.execute.first
+  end
+
+  private
+
+  def defendant_identifier
+    defendant.arrest_summons_number || defendant.national_insurance_number
+  end
+
+  def load_and_authorize_search
+    @search = Search.new(filter: 'defendant_reference', term: term)
+    authorize! :create, @search
+  end
+
+  def set_defendant_if_required
+    defendant
+  end
+
+  def term
+    @defendant_asn_or_nino || defendant_params[:id]
+  end
+
+  def add_defendant_case_breadcrumb
```

Comment:
I find this method in here to be a little odd - maybe we should just repeat this call in the relevant controllers and remove from this concern. The concern should be for one thing and one thing only which to me would be enabling search of a defendant as [mentioned above](https://github.com/ministryofjustice/laa-court-data-ui/pull/216/files#r454958421)

Reply:
I have moved the add_defendant_case_breadcrumb back into the two controllers but this is causing code climate to fail - should we just ignore this or attempt to refactor in a different way?

Reply:
ignore for now i'd say